src/components/Thought.tsx

```diff
@@ -426,6 +424,8 @@ const ThoughtContainer = ({
               publish={publish}
               simplePath={simplePath}
               thoughtId={thoughtId}
+              // debugIndex={debugIndex}
+              // depth={depth}
```

Comment:
Please never comment out code unless it is needed for expository purposes. If it is not necessary, remove it completely.

Reply:
I had planned to use these for color blocks of bullet so I commented it out, I wouldn't have left them otherwise. But if you want me to remove them I will, will just add the code back when adding the color blocks.

Reply:
Got it, sorry about that. I didn't realize it was intentional. Just leave a note for me so that I know next time.